src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -587,6 +596,22 @@ function PurchaseOrderUtility({ creating }) {
                                 disabled
                             />
                         </Grid>
+                        <Grid item xs={1}>
+                            {order.issuePartsToSupplier === 'Y' ? (
```

Comment:
I prefer the

 { order.issuePartsToSupplier === 'Y' && <Markup /> } 

syntax (over the ternary/conditional operator syntax) for these simple 'if something then render something ' style expressions. Just slightly less code - don't feel too strongly about it. s'up to you